resources/assets/less/variables.less

```diff
@@ -21,6 +21,7 @@
 @font-content: 'Noto Sans', sans-serif;
 @font-default: 'Exo 2', 'Helvetica Neue', Tahoma, Arial, 'Hiragino Kaku Gothic ProN', Meiryo, 'Microsoft YaHei', 'Apple SD Gothic Neo', sans-serif;
 @font-default-vi: Exo, 'Exo 2', 'Helvetica Neue', Tahoma, Arial, 'Hiragino Kaku Gothic ProN', Meiryo, 'Microsoft YaHei', 'Apple SD Gothic Neo', sans-serif;
+@font-default-zh: 'Exo 2', 'Helvetica Neue', Tahoma, Arial, 'Hiragino Kaku Gothic ProN', 'Microsoft YaHei', 'Apple SD Gothic Neo', system-ui, sans-serif;
```

Comment:
You'll probably want `Hiragino Sans GB` instead of `Hiragino Kaku Gothic ProN` for Chinese text.

Hiragino Sans GB
![image](https://user-images.githubusercontent.com/1694544/43696404-cac73be0-9978-11e8-81ab-e49448f104a0.png)
![43683438-9ef97d96-98c6-11e8-85c4-0658b24fc6f8](https://user-images.githubusercontent.com/1694544/43696253-1259a35e-9978-11e8-9fc4-b589accd9f83.png)

Hiragino Kaku Gothic ProN
![image](https://user-images.githubusercontent.com/1694544/43696414-d8ab1b32-9978-11e8-9bfb-76ede584c5a6.png)
![43683439-a8e22858-98c6-11e8-9bc2-b097e172fd60](https://user-images.githubusercontent.com/1694544/43696248-0f015594-9978-11e8-9a1b-17fbe1d51ec3.png)